src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.vb

```diff
@@ -147,7 +145,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 ' PERF: if allocations here become nuisance, consider caching the TupleTypeSymbol
                 '       in the type symbols that can actually be tuple compatible
-                tupleTypeSymbol = TupleTypeSymbol.Create(DirectCast(srcType, NamedTypeSymbol))
+                tupleTypeSymbol = tupleTypeSymbol.Create(DirectCast(srcType, NamedTypeSymbol))
```

Comment:
Will do. Thanks!

---
In reply to: [83752733](https://github.com/dotnet/roslyn/pull/14550#discussion_r83752733) [](ancestors = 83752733)